refinery/tool_manager/models.py

```diff
@@ -558,12 +559,10 @@ def _get_detailed_nodes_dict(self, node_uuid_list,
                 self.AUXILIARY_FILE_LIST: [
                     get_file_url_from_node_uuid(
                         uuid, require_valid_url=require_valid_urls
-                    ) for uuid in Node.objects.get(uuid=node["uuid"]).
-                    get_auxiliary_nodes()
+                    ) for uuid in
+                    Node.objects.get(uuid=node["uuid"]).get_auxiliary_nodes()
```

Comment:
Looks better but I'd also rename `get_auxiliary_nodes()` to `get_auxiliary_node_uuids()` to reflect the actual return value type.